donate.dd

```diff
@@ -0,0 +1,91 @@
+        to highly skilled students.)
+    $(LI organizes the yearly $(HTTP dconf.org, D conference).)
+    $(LI pays for hardware costs
+        (e.g. this website, $(LINK2 http://dlang.org/download.html, dmd downloads),
+        the $(LINK2 https://tour.dlang.org, D tour) and more))
+    $(LI is an accepted
+        $(HTTPS summerofcode.withgoogle.com/organizations/5078256051027968/,
+        Google Summer of Code organization) and hosted four projects during
+        the summer of 2016.
+    )
+)
+
+$(H3 How can I donate?)
+
+We welcome contributions from both organizations and individuals. Your help is highly appreciated.
+
+$(DONATE_ITEM Donate through PayPal, paypal,
+
+    Click the PayPal button to make a donation to the D Language Foundation via PayPal:
+    $(BR) $(BR)
+
+    <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
+    <input type="hidden" name="cmd" value="_s-xclick" />
+    <input type="hidden" name="hosted_button_id" value="K8PLNUP7276BQ" />
+    <input type="image" src="https://www.paypal.com/en_US/i/btn/btn_donate_LG.gif" border="0" name="submit" title="PayPal - The safer, easier way to pay online!" alt="Donate" />
+    <img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1" />
+    </form>
+)
+
+$(DONATE_ITEM Donate by credit card, credit-card,
+    Please use the PayPal button above to donate by credit card. You don't need a PayPal account to use the donation button.
+)
+
+$(DONATE_ITEM Electronic wire transfers or bank check, money,
+    Wire transfer information will be announced soon. In the meantime please use PayPal or $(LINK2 mailto:foundation@dlang.org, contact the D Language Foundation).
+)
+
+$(H3 If I donate financially, where do my donations go?)
+
+$(P The D Languages Foundation awards $(LINK2 http://dlang.org/blog/2016/12/05/the-d-language-foundations-scholarship-program, scholarships) to highly skilled students.
+Each $(DOLLAR)5 you donate contributes to approximately one hour of work by a talented graduate student, and
+these hours add quickly toward important projects that further the state of the art in the D
+programming language.)
+
+$(P If you have a specific target or project that you wish to directly support, please $(LINK2 mailto:foundation@dlang.org, contact the D Language Foundation).)
+
+$(P By the Foundation bylaws, its officers (Walter Bright, Andrei Alexandrescu and Ali Çehreli)
+    cannot get income from the Foundation, which preempts a variety of conflicts of interest.)
+
+$(H3 The D Language Foundation is a non-profit organization)
+
+$(P For US taxpayers, contributions to the D Language Foundation are tax-deductible.
+The D Language Foundation has been recognized as a public charity under section 501(c)(3) of the Internal Revenue Code.
+You should consult your tax adviser for rules as they apply to your situation.)
```

Comment:
Ah, I see you do have this now. You may want to move the public charity part up the page. Leave here: For US taxpayers, contributions to the D Language Foundation are tax-deductible. You should consult your tax adviser for rules as they apply to your situation.